pkg/handler/crud.go

```diff
@@ -108,6 +110,11 @@ func (c *crud) FindFlags(params flag.FindFlagsParams) middleware.Responder {
 			fmt.Sprintf("%%%s%%", strings.ToLower(*params.DescriptionLike)),
 		)
 	}
+	if params.Deleted != nil {
```

Comment:
we also need to check if deleted is true or false.

```suggestion
	if params.Deleted != nil && *params.Deleted {
```